tests/db_test.py

```diff
@@ -0,0 +1,21 @@
+import pytest
+import sqlalchemy
+from sqlalchemy import create_engine
+from sqlalchemy_utils import database_exists
+
+
+db_uri = 'postgresql://postgres:edison@127.0.0.1/edison'
+
+@pytest.fixture
+def engine():
+    return create_engine(db_uri)
```

Comment:
I'm still new to this but I'm pretty sure that every time you use this fixture in a test it will create a new engine.
This is something that you want ? and if so why ?
If you wish to create it once you could change pytest.fixture to pytest.fixture(scope='module').

Reply:
Ack